src/app/screens/SignUp/components/CustomInput/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+interface Props extends React.HTMLProps<HTMLInputElement> {
+  inputRef?: (ref: HTMLInputElement) => void;
+  error?: string;
+}
+
+export default function CustomInput({ inputRef, error, ...props }: Props) {
+  return (
+    <>
+      <input {...props} ref={inputRef} className={styles.customInputs} />
```

Comment:
input has label, label should be here too as optional, because label is not always present